scripts/build.ts

```diff
@@ -19,7 +24,12 @@ fileReplace('dist/es/Constants.js', '__STANZAJS_VERSION__', Pkg.version);
 
 Child('npm run compile:rollup');
 
-Child('mkdir dist/npm');
+if (!FS.existsSync("dist")) {
+    FS.mkdirSync("dist");
+}
+if (!FS.existsSync("dist/npm")) {
+    FS.mkdirSync("dist/npm");
+}
```

Comment:
Why, though? The rest of the script file relies on gnutils anyways, might as well write it as `mkdir -p dist/npm`

Reply:
Because I'm running windows right now and I would rather replace this entire file with `fs` calls than get that set up and make it a dependency. It's completely unnecessary to call the shell here.